app/assets/stylesheets/application.scss

```diff
@@ -2,6 +2,27 @@ $govuk-image-url-function: "image-url";
 $govuk-font-url-function: "font-url";
 
 @import "../../../node_modules/govuk-frontend/govuk/all";
+
+
+.govuk-header__content,
+.govuk-header__logo {
+  @include govuk-media-query($from: desktop) {
+    width: 50%;
```

Comment:
out of curiosity, where is this 50% coming from?

Reply:
this is to make the banner fit on one line, it's adapted from https://github.com/alphagov/product-page-example/blob/transition-to-govuk-frontend/source/stylesheets/modules/_header.scss